### DISCUSSION TITLE
Modifying package dependencies doesn't seem to work for me?

### DISCUSSION THREAD

**Discussion by juliavdkris** [Morganamilo/paru#965] [Q&A]
Hi! I've been using paru for years now, and really like it. But if I needed to change a package's dependencies, I've always just resorted to cloning the repo from the AUR myself, editing it and doing a `makepkg -si`. I recently discovered that paru has a `--fm` flag for editing files before installing, but I can't quite get it to work.

The package I'm trying to install is podman-desktop, which depends on nodejs-lts-gallium. I want to use the regular nodejs instead, and in my experience that actually works fine. I first tried `paru --fm code -S podman-desktop`, and changed the PKGBUILD and SRCINFO files. But when I proceed with the installation, it still tries to install nodejs-lts-gallium. I tried adding the `--savechanges` flag to commit the changes, but even then the same issue occurs.

Is this a bug, or am I doing something wrong? Any help is much appreciated!

**Comment by hariganti** [comment]
I have also found this issue.

Currently, OpenSCAD has an issue in it's PKGBUILD that prevents it from building successfully, so I need to edit the PKGBUILD to adjust the issue in the dependencies. However, even though `qt5-gamepad` is no longer required, it seems that this gets pulled before I have a chance to save the edits after reviewing the PKGBUILD, so `qt5-gamepad` gets pulled from the AUR regardless of what I want.

I am wondering if the dependencies are not fully determined by the PKGBUILD itself
